source/_integrations/imap.markdown

```diff
@@ -84,27 +37,24 @@ By default, this integration will count unread emails. By configuring the search
 * `FROM`, `TO`, `SUBJECT` to find emails in a folder (see [IMAP RFC for all standard options](https://tools.ietf.org/html/rfc3501#section-6.4.4))
 * [Gmail's IMAP extensions](https://developers.google.com/gmail/imap/imap-extensions) allow raw Gmail searches, like `X-GM-RAW "in: inbox older_than:7d"` to show emails older than one week in your inbox. Note that raw Gmail searches will ignore your folder configuration and search all emails in your account!
 
-#### Full configuration sample with search
-
-```yaml
-# Example configuration.yaml entry for gmail
-sensor:
-  - platform: imap
-    server: imap.gmail.com
-    port: 993
-    username: YOUR_USERNAME
-    password: YOUR_PASSWORD
-    search: FROM <sender@email.com>, SUBJECT <subject here>
-    # Or use X-GM-RAW search-term like this, to find unread emails from the last 7 days in your inbox
-    # search: 'X-GM-RAW "in: inbox newer_than:7d is:unread"'
+#### Config entry sample with search examples
```

Comment:
We do not use the "Config entry" term much in user-facing documentation or UIs. 
A user won't have an idea what this headline means.